web/client/epics/annotations.js

```diff
@@ -369,12 +369,13 @@ export default (viewer) => ({
         }),
     startDrawingMultiGeomEpic: (action$, store) => action$.ofType(START_DRAWING)
         .filter(() => store.getState().annotations.editing.features && !!store.getState().annotations.editing.features.length || store.getState().annotations.featureType === "Circle")
-        .switchMap( () => {
+        .switchMap( ({options: opts}) => {
```

Comment:
i think is quicker if you destructure directly here
```suggestion
        .switchMap( ({options: {geodesic}} = {options: {} }) => {
```